### PR TITLE
[FIXUP] passing wrong object for transforming refs

### PR DESCRIPTION
**Pull request by newbie95**
Added fixup for the [issue](https://github.com/BuilderIO/mitosis/issues/1150)

### CODE REVIEW

**Comment by samijaber** [comment]
please create pull requests using our pull request template. it includes instructions and steps that you need to follow for your PR to be mergable to the repo.

https://github.com/BuilderIO/mitosis/blob/main/.github/pull_request_template.md

**Review by samijaber — CHANGES_REQUESTED** [review]
PR request template not followed: missing changeset and tests

File: packages/core/src/__tests__/test-generator.ts [hunk 1]
```diff
@@ -244,6 +247,7 @@ const BASIC_TESTS: Tests = {
   useTarget,
   signalsOnUpdate,
   getterState,
+  ref,
```
Comment (samijaber) [inline packages/core/src/__tests__/test-generator.ts:250]: looking at the snapshot tests updates, I see many existing snapshots getting fixed by your PR.

You can remove this new `lit-ref.raw.tsx` test and its snapshot: there are existing tests that cover your fix! 🙏🏽

**Review by samijaber — CHANGES_REQUESTED** [review]
Awesome, thanks! You can actually get rid of the new test: the existing snapshots cover your bugfix

File: packages/core/src/__tests__/test-generator.ts (hunk 1 above)
Comment (newbie95) [reply packages/core/src/__tests__/test-generator.ts]: That's great to hear! I've removed the new test cases.

**Pull request merged**